### PR TITLE
♻️ Move sequencing statistics to genomic file

### LINKED ISSUE #475: Consider removing fields from SE
I also would like to suggest that we these off of SE entirely and they will exist only on the appropriate GFs and/or RGs:
max_insert_size
mean_insert_size
mean_depth
total_reads
mean_read_length

But need some more input from the Bix team, so putting as an issue to discuss/decide

### PR DESCRIPTION
**Pull request by dankolbman**
Move following fields from `SequencingExperiment` to `GenomicFile`:
- `max_insert_size`
- `mean_insert_size`
- `mean_depth`
- `total_reads`
- `mean_read_length`

fixes #475

### CODE REVIEW

File: dataservice/api/genomic_file/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+prefix: `GF`
+
+Genomic Files track files related to the bioinformatics process such as `bam`
+`cram`, `gvcf`, and other files. Files may be related to a `Biospecimen`
+(eg: an aligned `cram`), others may not be directly related to `Biospecimen`,
+having been derived from some other file(s) (eg: a cohort `vcf`).
+Most all `Genomic Files` should be able to resolve to one or more `Biospecimens`
```
Comment (znatty22) [inline dataservice/api/genomic_file/README.md:7]: I think this last sentence is a little confusing. Don't all of our unharmonized and harmonized genomic files have links to a Biospecimen through the BiospecimenGenomicFile relation? Maybe its more like, most genomic files should be able to resolve to one or more specimens through the BiospecimenGenomicFile, and others can be resolved using the Task table. And then maybe add something about Task - Task is an executed bioinformatic process which produced Genomic Files from one or more input Genomic Files which are linked to existing Biospecimens.

**Review by znatty22 — COMMENTED** [review]
Looks good so far. Are you going to add the migration script as well?

**Comment by dankolbman** [comment]
Still needs more discussion in #475

File: dataservice/api/genomic_file/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+prefix: `GF`
+
+Genomic Files track files related to the bioinformatics process such as `bam`
```
Comment (parimalak) [inline dataservice/api/genomic_file/README.md:3]: missing `,`

**Review by parimalak — APPROVED** [review]

**Review by znatty22 — CHANGES_REQUESTED** [review]
Need more discussion on #475

**Comment by baileyckelly** [comment]
Per Allison - this is still valid and needs to be done. Moving to backlog as will not be tackled this sprint.

**Pull request closed without merging**